README.md

```diff
@@ -80,6 +80,58 @@ If all prerequisites have been followed, you can start KoalaBot with the followi
 $ python3 KoalaBot.py
 ```
 
+## Running KoalaBot with Docker
+Here are some example snippets to help you get started creating a container.
+
+### docker-compose (recommended, [click here for more info](https://docs.linuxserver.io/general/docker-compose))
+```yaml
+---
+version: "3.9"
+services:
+  transmission:
+    image: jaydwee/koalabot
+    container_name: KoalaBot
+    environment:
+      - DISCORD_TOKEN = bot_token
+      - BOT_OWNER = owner_user_id #optional
+      - ENCRIPTION = boolean #optional
```

Comment:
is this a typo for `ENCRYPTION`, and what does it do if you enable it?

Reply:
Yep, that is a typo, mb. Will fix when I get home.

It overrides whether encryption with sqlcipher is enabled or disabled. Enabled by default

Reply:
awesome, i think pending that change then everything else LGTM!